Source/SvgDocument.cs

```diff
@@ -461,46 +466,43 @@ public void Draw(Graphics graphics, SizeF? size)
                 throw new ArgumentNullException("graphics");

```

Comment:
What is this method used for ?
Not `SizeF` but `RectangleF` ?

Reply:
see #337

Reply:
@mrbean-bremen 

Thanks!!

I test following source code.
```cs
var imageSize = svgDoc.GetDimensions();

// same size
using (var image = new Bitmap((int)imageSize.Width, (int)imageSize.Height))
using (var graphics = Graphics.FromImage(image))
{
    var size = new SizeF(imageSize.Width, imageSize.Height);
    svgDoc.Draw(graphics, size);
}

// size up
using (var image = new Bitmap((int)imageSize.Width, (int)imageSize.Height))
using (var graphics = Graphics.FromImage(image))
{
    var size = new SizeF(imageSize.Width * 2, imageSize.Height * 2); 
    svgDoc.Draw(graphics, size);
}

// size down
using (var image = new Bitmap((int)imageSize.Width, (int)imageSize.Height))
using (var graphics = Graphics.FromImage(image))
{
    var size = new SizeF(imageSize.Width / 2, imageSize.Height / 2);
    svgDoc.Draw(graphics, size);
}
```
The results were all the same.

Reply:
ok, that doesn't seem right...